.github/workflows/ci.yml

```diff
@@ -22,6 +22,7 @@ jobs:
     if: "${{ github.event_name != 'push' || (github.event_name == 'push' && github.repository == 'nf-core/methylseq') }}"
     runs-on: ubuntu-latest
     strategy:
+      fail-fast: false
```

Comment:
@ewels do we still want this fail fast to false in the new CI ?

Reply:
looks like it will be handy to not kill all tests at once...so going to add this in